README.md

```diff
@@ -7,16 +7,20 @@ A PHP client for connecting to the Regalii API.
 After requiring `regalii/regaliator` in your `composer.json` file, you can use the class like so:
 
 ```php
-$regaliator = new Regaliator\Regaliator(
-  'https://test.casiregalii.com',
-  getenv('REGALII_API_KEY'),
-  getenv('REGALII_SECRET')
-);
+$configuration = new Regaliator\Configuration([
+  'version' => '3.1',
+  'api_host' => 'https://test.casiregalii.com',
```

Comment:
FYI on the Ruby client, we doesn't specify the protocol only the domain name, i.e: `test.casiregalii.com`.